.github/workflows/static-analysis.yml

```diff
@@ -65,3 +65,6 @@ jobs:
 
       - name: "Run Psalm"
         run: "vendor/bin/psalm --show-info=false --stats --output-format=github --threads=$(nproc)"
+
+      - name: "Run Rector"
+        run: "vendor/bin/rector --ansi --dry-run"
```

Comment:
Noted that `--dry-run` returns a non-zero exit code if changes are detected, which makes it suitable for CI.